packages/twenty-front/src/modules/object-record/hooks/useLazyFindManyRecords.ts

```diff
@@ -83,6 +90,20 @@ export const useLazyFindManyRecords = <T extends ObjectRecord = ObjectRecord>({
   const findManyRecordsLazy = useRecoilCallback(
     ({ set }) =>
       async () => {
+        if (!hasReadPermission) {
```

Comment:
why do we need to treat this edge case manually? can't we trust the backend response?